inc/azure_c_shared_utility/shared_util_options.h

```diff
@@ -15,6 +15,7 @@ extern "C"
         int port;
         const char* username;
         const char* password;
+		const char* protocol;
```

Comment:
>protocol [](start = 14, length = 8)

What values can be used here? It would be nice to have at least a comment with the possible values somewhere.